rotkehlchen/history/events/structures/eth2.py

```diff
@@ -207,9 +213,18 @@ def process(
             accounting: 'AccountingPot',
             events_iterator: Iterator['AccountingEventMixin'],  # pylint: disable=unused-argument
     ) -> int:
+        # this is to avoid a circular import.
+        from rotkehlchen.db.eth2 import DBEth2
```

Comment:
instead of doing this maybe you can adjust the process function to take kwargs and when it is a eth2 event you pass `DBEth2` from the accountant class? This avoids the import that is quite bad in functions in a loop